### DISCUSSION TITLE
system.io.abstractions monorepo has incomplete sourceurls

### DISCUSSION THREAD

**Discussion by mratwork-dev** [renovatebot/renovate#32615] [Suggest an Idea]
### Tell us more.

The definition of https://docs.renovatebot.com/presets-monorepo/#monoreposystemioabstractions is incomplete,

```
{
  "matchSourceUrls": [
    "https://github.com/System-IO-Abstractions/System.IO.Abstractions/"
  ]
}
```

Should be changed to: 
```
{
  "matchSourceUrls": [
    "https://github.com/System-IO-Abstractions/System.IO.Abstractions/",
    "https://github.com/TestableIO/System.IO.Abstractions"
  ]
}
```

Reason: nuget packages System.IO.* are also published as TestableIO.System.IO.*
Note, when you navigate to https://github.com/System-IO-Abstractions/System.IO.Abstractions that you are automatically redirected to https://github.com/TestableIO/System.IO.Abstractions

**Comment by rarkins** [comment]
PR welcome to add this URL

**Reply by mratwork-dev** [reply]
Ok, I thought I didn't have permissions to create a PR. Not sure how to do that, but I will see

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#code
